### PR TITLE
Update data.js

### PR DESCRIPTION
**Pull request by conrad96**

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -78,6 +78,25 @@ const pages = [
...
     ],
   },
+  {
+    name: 'Conrad Mugisha',
+    description:
+      'Software Developer. Lover of challenges',
+    url: 'https://aless.co/uses',
+    twitter: '@conradmugisha',
+    emoji: '😎',
+    country: 'UG',
```
Comment (palashmon) [inline src/data.js]: Please pull your country emoji code from https://github.com/wesbos/awesome-uses/blob/master/scripts/flags.js

File: src/data.js [hunk 2]
```diff
@@ -78,6 +78,25 @@ const pages = [
       'Blogger',
     ],
   },
+  {
+    name: 'Conrad Mugisha',
+    description:
+      'Software Developer. Lover of challenges',
+    url: 'https://aless.co/uses',
```
Comment (HugoDF) [inline src/data.js]: This is Alessia Bellisario's "uses" page I believe

Comment (conrad96) [reply src/data.js]: oh snap, forgot to edit that

File: src/data.js (hunk 1 above)
Comment (conrad96) [reply src/data.js]: okay. right (y)

File: src/data.js [hunk 3]
```diff
@@ -78,6 +78,25 @@ const pages = [
       'Blogger',
     ],
   },
+  {
+    name: 'Conrad Mugisha',
+    description:
+      'Software Developer. Lover of challenges',
+    url: 'https://github.com/conrad96',
```
Comment (HugoDF) [inline src/data.js]: This needs to be a "uses" URL, I think we're allowing Gist's containing your equipment & software if you don't have a site where you can add the page.

**Comment by HugoDF** [comment]
@conrad96 closing this for now, feel free to re-open this when you've got a "uses" page (doing it in a Gist is fine)

**Pull request closed without merging**